src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -202,92 +205,99 @@ private void LoadItems(PackageItemListViewModel selectedPackageItem, Cancellatio
 
             var currentLoader = _loader;
 
-            _joinableTaskFactory.Value.RunAsync(async () =>
-            {
-                await TaskScheduler.Default;
+            _joinableTaskFactory.Value.RunAsync(
```

Comment:
Personally, I prefer using rather than our own abstraction to help vs-threading analyzers to detect more problems. But, it seems you're just renaming the method from `LoadItems()` to RepopulatePackageList()`. Is that correct?

```suggestion
            ThreadHelper.JoinableTaskFactory.RunAsync(
```

Reply:
I did not change the JTF method here... and in fact, the ctor sets this to `NuGetUIThreadHelper.JoinableTaskFactory` - but that's irrelevant to my PR.

I didn't rename LoadItems, but instead extracted the delegate to a method so that I could associate FileAndForget with a method name.